contracts/deploy/06_deploy_TokenBridgeWithReinitialization.ts

```diff
@@ -0,0 +1,85 @@
+import { ethers, upgrades } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { tryVerifyContract, getDeployedContractAddress, getRequiredEnvVar } from "../common/helpers";
+import { TokenBridge__factory } from "contracts/typechain-types";
+import {
+  L2_MESSAGE_SERVICE_PAUSE_TYPES_ROLES,
+  L2_MESSAGE_SERVICE_UNPAUSE_TYPES_ROLES,
+  PAUSE_ALL_ROLE,
+  PAUSE_L1_L2_ROLE,
+  PAUSE_L2_L1_ROLE,
+  UNPAUSE_ALL_ROLE,
+  UNPAUSE_L1_L2_ROLE,
+  UNPAUSE_L2_L1_ROLE,
+  USED_RATE_LIMIT_RESETTER_ROLE,
+} from "contracts/common/constants";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const securityCouncilAddress = "0xcA11bde05977b3631167028862bE2a173976CA11";
+
+  const newRoleAddresses = [
+    { addressWithRole: securityCouncilAddress, role: USED_RATE_LIMIT_RESETTER_ROLE },
+    { addressWithRole: securityCouncilAddress, role: PAUSE_ALL_ROLE },
+    { addressWithRole: securityCouncilAddress, role: PAUSE_L1_L2_ROLE },
+    { addressWithRole: securityCouncilAddress, role: PAUSE_L2_L1_ROLE },
+    { addressWithRole: securityCouncilAddress, role: UNPAUSE_ALL_ROLE },
+    { addressWithRole: securityCouncilAddress, role: UNPAUSE_L1_L2_ROLE },
+    { addressWithRole: securityCouncilAddress, role: UNPAUSE_L2_L1_ROLE },
+  ];
```

Comment:
Could you use `generateRoleAssignments` function here to generate this mapping ? :)